### PR TITLE
Add the 'start', 'start:saleor-app-checkout' and 'start:storefront' s…

### LINKED ISSUE #422: Add a `start` script at the parent monorepo level
### Summary

While being at the top level monorepo directory for `react-storefront`, it would be handy to run the `pnpm start` command to launch its `storefront` and `saleor-app-checkout` apps.

At this moment that command fails because such `start` script does not exist for the project.

This is with the `react-storefront` sources of commit ID `28ed7e53916e107fb25f837b5093787de0c97426`.

The ultimate goal of this requirement is to have a convenient way to execute the `next start` command defined by the `start` script inside the files `apps/storefront/package.json` and `apps/saleor-app-checkout/package.json`, which are suitable for a production mode.

### Steps to reproduce

1. Go to your local `react-storefront` directory.
2. Run `pnpm build` and wait for it to finish.
3. Run `pnpm start`.
4. An error output like the following is generated:
    > saleor-storefront@0.0.0 start /projects/react-storefront
    > node server.js
    node:internal/modules/cjs/loader:958
      throw err;
      ^
    Error: Cannot find module '/projects/react-storefront/server.js'
        at Module._resolveFilename (node:internal/modules/cjs/loader:955:15)
        at Module._load (node:internal/modules/cjs/loader:803:27)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
        at node:internal/main/run_main_module:18:47 {
      code: 'MODULE_NOT_FOUND',
      requireStack: []
    }
    Node.js v18.8.0
     ELIFECYCLE  Command failed with exit code 1.

### PR DESCRIPTION
**Pull request by rodolfojcj**
Resolves #422.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +172,27 @@ You can also run only a specific app by running this command:
 pnpm dev --filter=saleor-app-checkout
 ```
 
+## Starting on your own production server
```
Comment (peelar) [inline README.md:175]:
```suggestion
## Starting production server
```

File: README.md [hunk 2]
```diff
@@ -172,6 +172,27 @@ You can also run only a specific app by running this command:
...
+
+After completing the development as per above, these commands are suitable to start the `storefront` and `saleor-app-checkout` applications in a production-like environment:
+
+```bash
+pnpm build
+pnpm start
+```
+
+Alternatively, you can also start only a specific application by running either of these commands:
```
Comment (peelar) [inline README.md:184]:
```suggestion
Alternatively, you can start a specific application by running either of these commands:
```

Comment (peelar) [inline README.md:184]:
```suggestion
Alternatively, you can start a specific application by running either of these commands:
```

File: README.md [hunk 3]
```diff
@@ -172,6 +172,27 @@ You can also run only a specific app by running this command:
 pnpm dev --filter=saleor-app-checkout
 ```
 
+## Starting on your own production server
+
+After completing the development as per above, these commands are suitable to start the `storefront` and `saleor-app-checkout` applications in a production-like environment:
```
Comment (peelar) [inline README.md:177]:
```suggestion
After completing the development, these commands are suitable to start the `storefront` and `saleor-app-checkout` applications in a production-like environment:
```

File: turbo.json [hunk 4]
```diff
@@ -59,6 +59,8 @@
     },
     "dev": {
       "cache": false
+    },
+    "start": {
```
Comment (witoszekdev) [inline turbo.json:63]: I think we should disable Turborepo caching just like in `dev` command:

```suggestion
    "start": {
      "cache": true
```

File: README.md (hunk 2 above)
Comment (mmiszy) [inline README.md]:
```suggestion
Alternatively, you can start a specific application by running either of these commands:
```

**Review by mmiszy — APPROVED** [review]

**Review by mmiszy — APPROVED** [review]

**Pull request merged**